articles/azure-cache-for-redis/quickstart-create-redis-enterprise.md

```diff
@@ -17,6 +17,8 @@ Azure Cache for Redis' Enterprise tiers provide fully integrated and managed [Re
 * Enterprise, which uses volatile memory (DRAM) on a virtual machine to store data
 * Enterprise Flash, which uses both volatile and non-volatile memory (NVMe or SSD) to store data.
 
+Both support redis 6.0, and some new features not yet available in the other tiers, including some redis modules.
```

Comment:
At some point, we decided to avoid .0 (point anything) in the docs just as it is in the UI. I'm being pedantic but I hope clear by adding this stuff.

```suggestion
Both Enterprise and Enterprise Flash support open-source Redis 6 and some new features not yet available in the Basic, Standard or Premium tiers, including some Redis modules.  
```
Also, can we be a bit more specific about the module?